src/subscriptions/TransactionFetcher.ts

```diff
@@ -55,15 +55,14 @@ export default class TransactionFetcher {
       const variables = {
         contractAddress: entityRecord.contractAddress,
         uts,
-        limit: this.queryLimit,
```

Comment:
It seems that we have 2 sources of truth for `this.queryLimit`. In `src/GraphQueries/FetchQueries.ts` we have hardcoded `first: 100` in all queries. Ideally even this `100` should come from a query param.

Reply:
Agreed. Fetch queries needs to be updated to render from limit parameter.